_topic_map.yml

```diff
@@ -734,20 +734,24 @@ Topics:
   File: administrator-cli-commands
   Distros: openshift-enterprise,openshift-origin
 #---
-#Name: Container-native Virtualization
-#Dir: cnv
-#Distros: openshift-enterprise
-#Topics:
+Name: Container-native Virtualization
```

Comment:
These are in so that netlify will build the draft, but I'll pull them before I merge.